fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/provider/foundationdb/FDBDatabaseRunner.java

```diff
@@ -63,7 +63,7 @@
  *
  * @see FDBDatabase
  */
-@API(API.Status.MAINTAINED)
+@API(API.Status.INTERNAL)
```

Comment:
I think we still want the class to be `MAINTAINED`, but the constructor should be `INTERNAL`. Like, I think it's still expected that someone might use the class (or interface after #638), but they shouldn't be creating it themselves.

Reply:
Fixed (I forgot to reply yesterday)